airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryGcsOperations.java

```diff
@@ -84,7 +87,15 @@ public String getStagingFullPath(final String datasetId, final String stream) {
   public void createSchemaIfNotExists(final String datasetId, final String datasetLocation) {
     if (!existingSchemas.contains(datasetId)) {
       LOGGER.info("Creating dataset {}", datasetId);
-      BigQueryUtils.getOrCreateDataset(bigQuery, datasetId, datasetLocation);
+      try {
+        BigQueryUtils.getOrCreateDataset(bigQuery, datasetId, datasetLocation);
+      } catch (BigQueryException bqe) {
```

Comment:
Super nit: since we're only catching one exception type, it doesn't seem necessary to have an Exception value that differs from the rest of the code 

```java
BigQueryException e
```

That said, it's not blocking. It's mostly to keep consistency across our code base.

Also in the super class of `BigQueryException`, `BaseServiceException` there's a method `getReason()` that might be more usage to return back to the user for how come something caused this exception